src/index.html

```diff
@@ -8,16 +8,13 @@
   </head>
   <body>
     <form>
-      <!-- TODO: Fix Logo Rendering Issue-->
-      <img src="/logo.png" />
+      <img src="logo.svg" />
       <span id="success-message" hidden="true">Success!</span>
       <span id="error-message" hidden="true">Invalid Credentials</span>
```

Comment:
Great job updating the password input to hide password visibility! Keep up the good work.